src/main/java/net/pms/formats/FormatFactory.java

```diff
@@ -38,30 +38,58 @@
 	 * Initial list of known formats.
 	 */
 	private static final Format[] FORMATS = new Format[] {
+		new AC3(),
+		new ADPCM(),
+		new ADTS(),
+		new AIFF(),
+		new ASS(),
+		new ATRAC(),
+		new AU(),
 		new BMP(),
+		new DSDAudio(),
+		new DTS(),
 		new DVRMS(),
+		new EAC3(),
 		new FLAC(),
 		new GIF(),
 		new ISO(),
```

Comment:
@Nadahar What about IDX and SUP ?
Yes i know that .sup have some issue with FFMpeg but i already submit an issue on that to FFMpeg.